setup.py

```diff
@@ -59,6 +59,7 @@
         "jsonstreams>=0.6.0",
         "semver>=2.13.0",
         "Pillow>=9.2.0",
+        "python-editor==1.0.4",
```

Comment:
Leaving a note for future reference: license is Apache 2.0, which is compatible.